src/components/menu/index.js

```diff
@@ -1,98 +1,98 @@
+import { useSelector, useDispatch } from "react-redux";
+import { Icon } from "../../utils/general";
+import "./menu.scss";
 
-import * as Actions from '../../actions';
+import * as Actions from "../../actions";
 
-export const ActMenu = ()=>{
-  const menu = useSelector(state=>state.menus);
-  const {abpos, isLeft} = useSelector(state=>{
+export const ActMenu = () => {
+  const menu = useSelector((state) => state.menus);
+  const { abpos, isLeft } = useSelector((state) => {
     var acount = state.menus.menus[state.menus.opts].length;
     var tmpos = {
-      'top': state.menus.top,
-      'left': state.menus.left
-    }, tmpleft = false;
+        top: state.menus.top,
+        left: state.menus.left,
+      },
+      tmpleft = false;
 
     var wnwidth = window.innerWidth,
-        wnheight = window.innerHeight;
+      wnheight = window.innerHeight;
 
     var ewidth = 312,
-        eheight = acount*28;
+      eheight = acount * 28;
 
-    tmpleft = (wnwidth - tmpos.left) > 504;
-    if(wnwidth - tmpos.left < ewidth){
+    tmpleft = wnwidth - tmpos.left > 504;
+    if (wnwidth - tmpos.left < ewidth) {
       tmpos.right = wnwidth - tmpos.left;
       tmpos.left = null;
     }
 
-    if(wnheight - tmpos.top < eheight){
+    if (wnheight - tmpos.top < eheight) {
       tmpos.bottom = wnheight - tmpos.top;
       tmpos.top = null;
     }
 
     return {
       abpos: tmpos,
-      isLeft: tmpleft
+      isLeft: tmpleft,
     };
-  })
+  });
 
   const dispatch = useDispatch();
 
-  const clickDispatch = (event)=>{
+  const clickDispatch = (event) => {
     event.stopPropagation();
     var action = {
       type: event.target.dataset.action,
-      payload: event.target.dataset.payload
+      payload: event.target.dataset.payload,
     };
 
-    if(action.type){
-      if(action.type!=action.type.toUpperCase()){
-        Actions[action.type](action.payload,menu);
-      }else{
+    if (action.type) {
+      if (action.type != action.type.toUpperCase()) {
+        Actions[action.type](action.payload, menu);
+      } else {
         dispatch(action);
       }
-      dispatch({type: "MENUHIDE"});
+      dispatch({ type: "MENUHIDE" });
     }
-  }
+  };
 
-  const menuobj = (data)=>{
+  const menuobj = (data) => {
     var mnode = [];
-    data.map(opt=>{
-      if(opt.type=="hr"){
+    data.map((opt) => {
+      if (opt.type == "hr") {
         mnode.push(<div className="menuhr"></div>);
-      }else{
-        mnode.push(<div className="menuopt" data-dsb={opt.dsb}
-          onClick={clickDispatch} data-action={opt.action}
-          data-payload={opt.payload}>
-          <div className="spcont">
-            {opt.icon?<Icon src={opt.icon} width={16}/>:null}
+      } else {
+        mnode.push(
+          <div className="menuopt" data-dsb={opt.dsb} onClick={clickDispatch} data-action={opt.action} data-payload={opt.payload}>
+            <div className="spcont">{opt.icon ? <Icon icon={opt.icon} width={16} /> : null}</div>
```

Comment:
the only change I made in this file is this, rest is a gift from prettier

Reply:
i.e. changing `<Icon src ` to `<Icon icon`

Reply:
How does that work for non svg icons? like terminal and about

Reply:
> How does that work for non svg icons? like terminal and about

it doesn't work for non svg icons so I got svg versions of them too :)

Reply:
I think its better to add a type property (only for non-img, like svg) in menu.js. because we can like always have svg version of icon available

Reply:
> I think its better to add a type property (only for non-img, like svg) in menu.js. because we can like always have svg version of icon available

sorry can you tell what exactly to do (i am a noob and don't even know enogh JS to creact a todo list)

Reply:
dont underestimate yourself, I will merge this in the test branch and I will make the require changes. Its hardly few if/else.